translation: initial chinese translation

hedyhli opened the pull request:
I'll try to force push into one commit at the end, but preferably please squash merge.

hedyhli committed:
translation: initial chinese translation

translation: special interest groups for zh

zh: translate all of goals

zh: translate all values

zh: translate soft and hard forks

zh: translate roadmap org and infra

zh: translate roadmap alignment stage

zh: misc fixes

zh: fix phrase for soft/hard forks

Co-authored-by: Gen <buzhybrid@gmail.com>

zh: adjust wording of root.description

zh: adjust SIGs description

zh: fix terminology use for 'upstream'

hedyhli commented:
All components of the front page has now been translated! I'll proofread and review the entire page soon, as a whole.

GenShibe commented:
imo `硬代码分支` should become `硬分支` and `软代码分支` should become `软分支, as it's not a "hard/soft code fork" but a "hard/soft fork"

(i ran this past @isabelroses and she agrees sooooo idk :p)

hedyhli commented:
It's not immediately clear IMO some sort of "divergence" is to do with repo/code, since "fork" in english is a technical term, but that said I don't actually use the chinese locale on software forge interfaces, so I'll definitely incorporate that suggestion to be more concise.

hedyhli commented:
Updated technical terminology to be consistent with that of [GitHub's docs](https://docs.github.com/zh/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks), rebased and squashed into one commit. (Failing CI due to #21)

Would appreciate reviews if there are native speakers here, but otherwise should be ready to go.

hedyhli commented on public/locales/zh/translations.json:
```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Nix 生态链的一个替换方案",
```
`Nix 生态链的另外一个选择`: (a more "direct" translation) "Another option" / "An alternative"
`Nix 生态链的一个替换方案`: (DWIM-style, what I've taken for the english version to actually mean) "A substitute plan"

hedyhli commented on public/locales/zh/translations.json:
```diff
@@ -0,0 +1,49 @@
+	"roadmap.organization.title": "组织",
+	"roadmap.organization.1": "我们将把部分的代码包的代码库提取出，由特殊兴趣小组来更方便的管理各自的流程。例如，Python 的代码包将被转出成独立的代码库，由Python 兴趣小组来管理。",
+	"roadmap.organization.2": "Flakes 将以当下的状态以 v0 版本来被稳定化。虽然这不一定最佳选择，但由于它的市面用途太广泛，不容易被轻易改动或去除，导致生态链的损坏。今后的改进将会专门被 Flakes 兴趣小组来领导。",
+	"roadmap.organization.3": "`aux` 的命令行工具将采纳更人性化管理代码包和系统的改进。我们也将增加格外的命令，如 `aux system switch` 及 `aux system build` 来把系统就职及管理的任务变得更方便。",
```
"make onboarding and ongoing maintenance easier" -> `系统就职及管理的任务变得更方便`

`就职` sounds a lot more like onboarding for role/positions than "integrating a new system" in this case, but I couldn't find a good alternative phrase here.

hedyhli commented on public/locales/zh/translations.json:
```diff
@@ -0,0 +1,49 @@
+	"roadmap.infra.2": "帮助文档的相关材料将作为持续集成流程的一部分包含在内，并将配置持续部署以使其保持最新状态，从而使贡献者们能够更轻松的创建科普材料。",
+	"roadmap.infra.3": "我们还会创建软件包和选项的搜索的基础设施。",
+	"roadmap.alignment.title": "对齐阶段",
+	"roadmap.alignment.1": "此时，我们做到了。我们创建了 Nix 生态系统的一个可持续的、独立的分支。初步工作已经完成，我们将开始治理的首次轮换过程，进行选举及关于如何运行操作的轮换科普。",
```
IMO this sounds more like "We did" rather than "We did it" but my attempts to rectify this had ended up becoming too verbose and added extraneous, unintended connotations.

On second thought this seems fine.

hedyhli reviewed (commented):
A few comments for consideration in case anyone has ideas for improvement.
